esp/services/ws_sql/SQL2ECL/ECLEngine.cpp

```diff
@@ -500,7 +500,10 @@ void ECLEngine::generateSelectStruct(HPCCSQLTreeWalker * selectsqlobj, IProperti
         }
         else if (col->getExpType() == Function_ExpressionType)
         {
-            SQLFunctionExpression * funcexp = dynamic_cast<SQLFunctionExpression *>(col);
+            SQLFunctionExpression * funcexp = static_cast<SQLFunctionExpression *>(col);
+            if (!funcexp)
```

Comment:
This test is spurious (and coverity will complain about it). funcexp cannot be null following a static-cast unless col was null - and if col was null then you cored two lines ago.